dotnetcore3.1/cookiecutter-aws-sam-quick-start-s3-dotnet/{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,149 @@
+# {{ cookiecutter.project_name }}
+
+This project contains source code and supporting files for a serverless application that you can deploy with the SAM CLI. It includes the following files and folders.
+
+- src - Code for the application's Lambda function.
+- events - Invocation events that you can use to invoke the function.
+- test - Unit tests for the application code. 
+- template.yaml - A template that defines the application's AWS resources.
+
+The application uses several AWS resources, including Lambda functions and an API Gateway API. These resources are defined in the `template.yaml` file in this project. You can update the template to add AWS resources through the same deployment process that updates your application code.
```

Comment:
The content is misleading, there is a function and S3 bucket in the template, but here we mention there is an API Gateway, can we fix the text?

Reply:
@mndeveci  nice catch